### ISSUE TITLE
About the loss in example reddit.py

### ISSUE THREAD

**Issue by hkn61** [rusty1s/pytorch_geometric#2882]
## ❓ Questions & Help

Hi! When I was running (https://github.com/rusty1s/pytorch_geometric/blob/master/examples/reddit.py), I found that although the accuracy was high, the training loss was increasing in general.

I tried to lower the learning rate and lower the probability of dropout, but the training loss was still increasing in general during training. What's the reason for that? Thanks for your help.

Here is the training output of reddit.py without any change of the code:
(https://user-images.githubusercontent.com/71131727/126336257-f4ef0cc9-58c1-4610-b66f-acf2f633421c.jpg)

**Comment by rusty1s** [comment]
I think this mostly denotes that training has converged, and updates are jumping around a local minimum. There is also a high variance in training due to sampling of edges and mini-batch processing, which might explain these jumps in loss values.

**Comment by hkn61** [comment]
I see. Thank you for your answer!
